util/execdetails/execdetails.go

```diff
@@ -53,60 +53,75 @@ type CommitDetails struct {
 	TxnRetry          int
 }
 
+const (
+	// ProcessTimeStr means the time of all coprocessor process.
+	ProcessTimeStr = "Process_time"
+	// WaitTimeStr means the time of all coprocessor wait.
+	WaitTimeStr = "Wait_time"
+	// BackoffTimeStr means the time of all back-off.
+	BackoffTimeStr = "Backoff_time"
+	// RequestCountStr means the request count.
+	RequestCountStr = "Request_count"
+	// TotalKeysStr means the total scan keys.
+	TotalKeysStr = "Total_keys"
+	// ProcessedKeysStr means the total processed keys.
+	ProcessedKeysStr = "Processed_keys"
+)
+
 // String implements the fmt.Stringer interface.
 func (d ExecDetails) String() string {
 	parts := make([]string, 0, 6)
 	if d.ProcessTime > 0 {
-		parts = append(parts, fmt.Sprintf("process_time:%vs", d.ProcessTime.Seconds()))
+		parts = append(parts, fmt.Sprintf("Process_time: %v", d.ProcessTime.Seconds()))
```

Comment:
I have try to use this name. But pt-query-digest have some problem cause by the name length.
```shell
▶ cat tidb.log
# Time: 2019-02-12-19=33=56.571953 +0800
# Txn_start_ts: 406315658548871171
# User: root@127.0.0.1
# Conn_ID: 6
# Query_time: 4.895492
# Sum_process_time: 0.161 Sum_wait_time: 0.2 Sum_backoff_time: 0.1 Request_count: 1 Total_keys: 100001 Processed_keys: 100000
# DB: test
# Is_internal: false
select * from t;
▶ pt-query-digest  tidb.log

# 60ms user time, 10ms system time, 19.15M rss, 175.60M vsz
# Current date: Thu Mar  7 18:00:28 2019
# Hostname: localhost.localdomain
# Files: tidb.log
# Overall: 1 total, 1 unique, 0 QPS, 0x concurrency ______________________
# Attribute          total     min     max     avg     95%  stddev  median
# ============     ======= ======= ======= ======= ======= ======= =======
# Exec time             5s      5s      5s      5s      5s       0      5s
# Query size            15      15      15      15      15       0      15
# Conn ID                6       6       6       6       6       0       6
# Processed ke      97.66k  97.66k  97.66k  97.66k  97.66k       0  97.66k
# Request coun           1       1       1       1       1       0       1
# Sum backoff        100ms   100ms   100ms   100ms   100ms       0   100ms
# Sum process        161ms   161ms   161ms   161ms   161ms       0   161ms
# Sum wait tim       200ms   200ms   200ms   200ms   200ms       0   200ms
# Total keys        97.66k  97.66k  97.66k  97.66k  97.66k       0  97.66k
# Txn start ts     360.88P 360.88P 360.88P 360.88P 360.88P       0 360.88P

# Profile
# Rank Query ID                           Response time Calls R/Call V/M
# ==== ================================== ============= ===== ====== =====
#    1 0x9F683436FE68935AF28708D8F2920792 4.8955 100.0%     1 4.8955  0.00 SELECT t

# Query 1: 0 QPS, 0x concurrency, ID 0x9F683436FE68935AF28708D8F2920792 at byte 0
# This item is included in the report because it matches --limit.
# Scores: V/M = 0.00
# Attribute    pct   total     min     max     avg     95%  stddev  median
# ============ === ======= ======= ======= ======= ======= ======= =======
# Count        100       1
# Exec time    100      5s      5s      5s      5s      5s       0      5s
# Query size   100      15      15      15      15      15       0      15
# Conn ID      100       6       6       6       6       6       0       6
# Processed ke 100  97.66k  97.66k  97.66k  97.66k  97.66k       0  97.66k
# Request coun 100       1       1       1       1       1       0       1
# Sum backoff  100   100ms   100ms   100ms   100ms   100ms       0   100ms
# Sum process  100   161ms   161ms   161ms   161ms   161ms       0   161ms
# Sum wait tim 100   200ms   200ms   200ms   200ms   200ms       0   200ms
# Total keys   100  97.66k  97.66k  97.66k  97.66k  97.66k       0  97.66k
# Txn start ts 100 360.88P 360.88P 360.88P 360.88P 360.88P       0 360.88P
# String:
# DB           test
# Is internal  false
# Time         2019-02-12-19=33=56.571953
# User         root@127.0.0.1
# Query_time distribution
#   1us
#  10us
# 100us
#   1ms
#  10ms
# 100ms
#    1s  ################################################################
#  10s+
# Tables
#    SHOW TABLE STATUS LIKE 't'\G
#    SHOW CREATE TABLE `t`\G
# EXPLAIN /*!50100 PARTITIONS*/
select * from t\G
```
As you can see, pt-query-digest have truncate name, seem the max name length is 12.

Reply:
Have you tried to enlarge the `max-line-length` option?
```
./pt-query-digest --help
...
  --max-line-length=i          Trim lines to this length. 0=Do not trim lines (
                               default 74)
```

Reply:
I just tried, it didn't work.